### ISSUE TITLE
MMS import fails for narrow intervals

### ISSUE THREAD

**Issue by dstansby** [heliopython/heliopy#697]
The following fails, even though there is data available in this interval:
```python
from datetime import datetime
from heliopy.data import mms

starttime = datetime(2017, 7, 11, 22, 30)
endtime = datetime(2017, 7, 11, 22, 36)

data = mms.fpi_dis_moms(1, 'fast', starttime, endtime)
print(data.data)

import matplotlib.pyplot as plt
fig, ax = plt.subplots()
ax.plot(data.data['mms1_dis_tempperp_fast'])
plt.show()
```

**Comment by nicolasaunai** [comment]
```python
bgsm =  mms.fpi_dis_moms(1, 'brst', datetime.datetime(2015, 10, 16, 13, 5, 24), datetime.datetime(2015, 10, 16, 13, 6, 29))
```

results in the following error

```
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-268-f74a942184fc> in <module>
----> 1 bgsm =  mms.fpi_dis_moms(1, 'brst', datetime.datetime(2015, 10, 16, 13, 5, 24), datetime.datetime(2015, 10, 16, 13, 6, 29))

/opt/venv/lib/python3.7/site-packages/heliopy/data/mms.py in fpi_dis_moms(probe, mode, starttime, endtime)
    190 def fpi_dis_moms(probe, mode, starttime, endtime):
    191     return download_files(probe, 'fpi', mode, starttime, endtime,
--> 192                           product_string='dis-moms')
    193 
    194 

/opt/venv/lib/python3.7/site-packages/heliopy/data/mms.py in download_files(probe, instrument, data_rate, starttime, endtime, verbose, product_string, warn_missing_units)
    163                         remote_base_url, download_func, processing_func,
    164                         starttime, endtime,
--> 165                         warn_missing_units=warn_missing_units)
    166 
    167 

/opt/venv/lib/python3.7/site-packages/heliopy/data/util.py in process(dirs, fnames, extension, local_base_dir, remote_base_url, download_func, processing_func, starttime, endtime, try_download, units, processing_kwargs, download_info, remote_fnames, warn_missing_units)
    192 
    193     # Loaded all the data, now filter between times
--> 194     data = timefilter(data, starttime, endtime)
    195     data = data.sort_index()
    196 

/opt/venv/lib/python3.7/site-packages/heliopy/data/util.py in timefilter(data, starttime, endtime)
    397     if len(data) == 0:
    398         raise RuntimeError(
--> 399             'No data available between {} and {}'.format(starttime, endtime))
    400     if isinstance(data, list):
    401         data = pd.concat(data)

RuntimeError: No data available between 2015-10-16 13:05:24 and 2015-10-16 13:06:29
```

It appears to work only when the end time is the next day.
Note this fails even though I have all CDFs on disk.

**Comment by argallmr** [comment]
The example provided by @nicolasaunai now works (but generates a lot of units warnings)

```
from heliopy.data import mms
from datetime import datetime
bgsm =  mms.fpi_dis_moms(1, 'brst', datetime(2015, 10, 16, 13, 5, 24), datetime(2015, 10, 16, 13, 6, 29))
>>> type(bgsm)
<class 'sunpy.timeseries.timeseriesbase.GenericTimeSeries'>
>>> bgsm.meta
|-------------------------------------------------------------------------------------------------|
|TimeRange                  | Columns         | Meta                                              |
|-------------------------------------------------------------------------------------------------|
|2015-10-16 13:05:24.134633 | mms1_dis_errorf |                                                   |
|            to             | mms1_dis_compre |                                                   |
|2015-10-16 13:06:28.935077 | mms1_dis_stepta |                                                   |
|                           | mms1_dis_energy |                                                   |
|                           | mms1_dis_energy |                                                   |
|                           | ...             |                                                   |
|-------------------------------------------------------------------------------------------------|
```

**Comment by dstansby** [comment]
Fixed in 0.9
